src/components/Footer/index.js

```diff
@@ -4,21 +4,18 @@ import React, { Component } from 'react';
 import styles from './styles';
 
 /* authors */
-const AUTHORS = ['anorudes', 'keske'];
+const AUTHORS = [
+  {
+    id: 1,
+    name: 'anorudes'
+  }, {
+    id: 2,
+    name: 'keske'
+  }
+];
 
 export class Footer extends Component {
   render() {
-    function renderGitHubFollowButton(user) {
-      return (
-        <a className="github-button" href={`https://github.com/${user}`}
-           data-count-href={`/${user}/followers`}
-           data-count-api={`/users/${user}#followers`}
-           data-count-aria-label="# followers on GitHub"
-           aria-label={`Follow @${user} on GitHub`}>
-          @{user}
-        </a>
-      );
-    }
```

Comment:
This function no longer necessary, but used all the jsx in the new map function below, changing only the string template contents.